cypress/support/step_definition/teams/commonTeamsSteps.spec.js

```diff
@@ -32,4 +34,36 @@ Then('I can see the checkbox for messenger in a team', () => {
 
 Then('I can see the chat in team', () => {
   teamsCommon.canSeeTeamChat()
+})
```

Comment:
Other methods with same content we named "see..." so here "seeTeamChat" would be enough I think.

Reply:
this method is written/implemented for different test before, but I can change in this PR